app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
@zachlatta Could you drop the slash command logic you've worked on so far into a PR so I can combine it with what I've worked on so far?